staging/src/k8s.io/kubectl/pkg/cmd/get/get_test.go

```diff
@@ -860,7 +860,7 @@ func TestGetSortedObjects(t *testing.T) {
 	cmd := NewCmdGet("kubectl", tf, streams)
 	cmd.SetOutput(buf)
 
-	// sorting with metedata.name
+	// sorting with metadata.name
```

Comment:
Saw this PR and fixed some spelling mistake since I am doing the `get.go`

https://github.com/kubernetes/kubernetes/pull/101996